wolfssl/wolfcrypt/settings.h

```diff
@@ -2033,7 +2025,7 @@ extern void uITRON4_free(void *p) ;
         #undef HAVE_ECC_KEY_IMPORT
         #define HAVE_ECC_KEY_IMPORT
     #endif
-    #ifndef NO_ECC_KEY_EXPORT
+    #if !defined(NO_ECC_KEY_EXPORT) && !defined(NO_BIG_INT)
```

Comment:
This is curious logic. Perhaps add a comment as to why `!defined(NO_BIG_INT)` is now required for `HAVE_ECC_KEY_EXPORT` ?